packages/node/src/methods/app-instance/get-free-balance/controller.ts

```diff
@@ -20,14 +28,23 @@ export default class GetFreeBalanceController extends NodeController {
     params: Node.GetFreeBalanceStateParams
   ): Promise<Node.GetFreeBalanceStateResult> {
     const { store } = requestHandler;
-    const { multisigAddress } = params;
+    const { multisigAddress, tokenAddress } = params;
 
     if (!multisigAddress) {
       Promise.reject(NO_STATE_CHANNEL_FOR_MULTISIG_ADDR);
     }
 
+    const tokenAddr = tokenAddress ? tokenAddress : AddressZero;
```

Comment:
Doesn’t need to be a variable, just place the value in the lookup in fbState. Also clearer variable names like freeBalanceState

Reply:
Only return at end of function

Reply:
This doesn't make sense. If a token address is specified but it's not in the free balance, just placing the conditional operator inside the lookup the returned value is undefined.